symlink/fs_unix_test.go

```diff
@@ -53,8 +53,8 @@ func testSymlink(tmpdir, path, expected, scope string) error {
 }
 
 func TestFollowSymlinkAbsolute(t *testing.T) {
-	tmpdir, cleanup := mkTempDir(t)
-	defer cleanup()
+	tmpdir := mkTempDir(t)
```

Comment:
Why don't we use `t.TempDir()` here? https://golang.org/pkg/testing/#T.TempDir

Ahh, this is because they are only supported since 1.15, so we need to wait at least until 1.14 is no longer supported.

Makes sense to add a TODO maybe?

Reply:
Ah, yes, can add a TODO

(didn't know about the `t.TempDir()` until you mentioned it recently. I  should catch up more on new features 😂)

Reply:
Can't use this shiny new t.TempDir() anyway until 1.14 goes out of fashion...